### PR TITLE
Review of Portal Getting Started Guide

### PR DESCRIPTION
**Pull request by suehighmoor**
Annual review of the Getting Started Guide for the UKCloud Portal
Verified contents against latest UI, updated screenshots to new style

### CODE REVIEW

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

File: articles/portal/ptl-gs.md [hunk 1]
```diff
@@ -142,17 +138,17 @@ The Portal navigation panel provides access to all the functionality available i
...
+- **Contacts** - Lists the user accounts with access to the current customer account. You can view a list of all user accounts (**All Contacts**) or just those with administration permissions (**Admin Contacts**). Portal administrators can create new users, reset other users' memorable words and change other users' Portal permissions (see [*Portal permissions*](ptl-ref-overview-permissions.md)).
 
-- **VMware Cloud** - Provides access to UKCloud for VMware compute services. Expand this option and then select the compute service with which you want to work (see the [*Getting Started Guide for UKCloud for VMware*](../vmware/vmw-gs.md)).
+- **VMware Cloud** - Provides access to UKCloud for VMware compute services. Expand this option and then select the compute service that want to work with (see the [*Getting Started Guide for UKCloud for VMware*](../vmware/vmw-gs.md)).
```
Comment (myoung-ukcloud) [inline articles/portal/ptl-gs.md:151]: I think this should say 'that you want to work with'

Comment (suehighmoor) [reply articles/portal/ptl-gs.md]: It should indeed! Thanks for spotting. All fixed.

**Review by myoung-ukcloud — APPROVED** [review]
LGTM

**Pull request merged**